discovery_engine_core/core/src/stack/filters/semantic.rs

```diff
@@ -317,4 +328,39 @@ mod tests {
         assert_eq!(filtered[1].id, documents[1].id);
         assert_eq!(filtered[2].id, documents[2].id);
     }
+
+    #[test]
+    fn test_normalized_distance() {
+        fn new_doc(embedding: &[f32], secs: i64) -> Document {
+            Document {
+                smbert_embedding: Embedding::from(arr1(embedding)),
+                resource: NewsResource {
+                    date_published: NaiveDateTime::from_timestamp(secs, 0),
+                    ..NewsResource::default()
+                },
+                ..Document::default()
+            }
+        }
+
+        let documents = vec![
+            new_doc(&[0.1, -0.5, 0.5], 100),
```

Comment:
Same as above, without knowing what's supposed to happen, it's difficult to verify whether these are correct / good test cases :D